pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -161,6 +162,15 @@ func (c *Cluster) Start() error {
 			}
 			logger.Infof("deployment for mgr %s already exists. updating if needed", resourceName)
 
+			// During an upgrade, make sure the server_addr setting is cleared or else the prometheus
+			// service may not be able to bind to the pod ip anymore, as was the setting in 4.7 and prior.
+			// This is only needed for 4.8 since upgrades to later clusters will always use the default.
+			logger.Info("ensuring the prometheus server_port is unset to binding to the localhost by default")
```

Comment:
I wonder if `Debug` wouldn't be more appropriate since it will run on any reconcile. Or perhaps add more context to it like the upgrade scenario?

Reply:
It's only printed once per reconcile and it's only going to be in the 4.8 logs for the one release, so it seems fine